dipy/utils/parallel.py

```diff
@@ -29,16 +29,19 @@ def paramap(
     func_kwargs=None,
     **kwargs,
 ):
+    # FIXME: but several fitting functions return "extra" as well
```

Comment:
Is this still an issue? Does any of your tests address this?

Reply:
I added tests on `extra` in the past and it was committed... and then this parallel stuff was added afterwards without seeming to consider it... so I don't really think I should be the one to add these tests, especially after finding errors in that original parallelization implementation that were precisely caused by not testing properly.